src/api/cpp/filters.cpp

```diff
@@ -16,8 +17,20 @@ namespace af
 
 array medfilt(const array& in, const dim_t wind_length, const dim_t wind_width, const borderType edge_pad)
 {
+    return medfilt2(in, wind_length, wind_width, edge_pad);
```

Comment:
call `af_medfilt` like it did before. The C API will punt to the proper function.